### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by pedrocamargo**
I am adding a CI workflow for documentation to make sure the documentation builds

Items to clear:

- [x] Are we keeping GitHub actions? If so, we need to replace references to Travis with the corresponding ones for the new testing
- [ ] The documentation talks about requiring PEP8 adherence for all contributions to the repository. Are we going to black and enforce linting?

### CODE REVIEW

**Comment by pedrocamargo** [comment]
I replaced the static reference to one git hash left behind with something dynamic, which also warns if there is something weird at the time of building it (e.g. not having committed all changes when I built documentation locally)
image

File: doc/source/gettingstarted.rst [hunk 1]
```diff
@@ -68,11 +79,13 @@ Installing from cloned source code (recommended for developers)
...
 
 Troubleshooting:
- - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling ``<fast-trips-dir>``.
+ - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling
+   ``<fast-trips-dir>``.
 
 .. note::
  - Pandas 0.21.x has known issues, and it is not compatible with Fast-Trips.
- - Fast-trips currently uses a development version of the `Partridge <https://github.com/remix/partridge>`_ that is required in order to read unzipped GTFS files.
+ - Fast-trips currently uses a development version of the `Partridge <https://github.com/remix/partridge>`_ that is
```
Comment (janzill) [inline doc/source/gettingstarted.rst]: @pedrocamargo not true anymore, you updated, right?

File: doc/source/gettingstarted.rst [hunk 2]
```diff
@@ -68,11 +79,13 @@ Installing from cloned source code (recommended for developers)
 
 
 Troubleshooting:
- - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling ``<fast-trips-dir>``.
+ - *Optional* Set the ``PYTHONPATH`` environment variable to the location of your fast-trips repo, which we're calling
+   ``<fast-trips-dir>``.
 
 .. note::
  - Pandas 0.21.x has known issues, and it is not compatible with Fast-Trips.
```
Comment (janzill) [inline doc/source/gettingstarted.rst]: Let's remove this line? The requirements should be >=1.0 or something

**Review by janzill — APPROVED** [review]
partridge and pandas comments are outdated, looks good otherwise

File: doc/source/gettingstarted.rst (hunk 2 above)
Comment (pedrocamargo) [reply doc/source/gettingstarted.rst]: Great catch. Fixed

**Pull request merged**